ansible/vars/topo_m0.yml

```diff
@@ -0,0 +1,271 @@
+topology:
+  host_interfaces:
+    - 0
+    - 1
+    - 2
+    - 3
+    - 4
+    - 5
+    - 6
+    - 7
+    - 8
+    - 9
+    - 10
+    - 11
+    - 12
+    - 13
+    - 14
+    - 15
+    - 16
+    - 17
+    - 18
+    - 19
+    - 20
+    - 21
+    - 22
+    - 23
+    - 24
+    - 25
+    - 26
+    - 27
+    - 28
+    - 29
+    - 30
+    - 31
+    - 32
+    - 33
+    - 34
+    - 35
+    - 36
+    - 37
+    - 38
+    - 39
+    - 40
+    - 41
+    - 42
+    - 43
+    - 44
+    - 45
+  VMs:
+    ARISTA01M1:
+      vlans:
+        - 46
+      vm_offset: 0
+    ARISTA02M1:
+      vlans:
+        - 47
+      vm_offset: 1
+    ARISTA03M1:
+      vlans:
+        - 48
+      vm_offset: 2
+    ARISTA04M1:
+      vlans:
+        - 49
+      vm_offset: 3
+    ARISTA05MX:
+      vlans:
+        - 50
+      vm_offset: 4
+    ARISTA06MX:
+      vlans:
+        - 51
+      vm_offset: 5
+  DUT:
+    vlan_configs:
+      default_vlan_config: one_vlan_a
+      one_vlan_a:
```

Comment:
I think we can keep `one_vlan_a` here only.

@wangxin Are `two_vlan_a` / `three_vlan_a`/... required for each topo? Have them might be useful for some general usage... but I have no idea on which test cases are relay on these topos.

Reply:
After discussing with @Blueve and @wangxin , keep one_vlan_a here only.